packages/react-server/src/ReactFizzServer.js

```diff
@@ -4111,14 +4168,26 @@ function retryRenderTask(
 
     const errorInfo = getThrownInfo(task.componentStack);
     task.abortSet.delete(task);
-    segment.status = ERRORED;
-    erroredTask(
-      request,
-      task.blockedBoundary,
-      x,
-      errorInfo,
-      __DEV__ && enableOwnerStacks ? task.debugTask : null,
-    );
+
+    if (x === AbortSigil) {
```

Comment:
If we abort sync in a resume, this can happen too right? Therefore you need to handle the same thing in `retryReplayTask`.

Reply:
done